buildconfig/CMake/Packaging/mantidpython.in

```diff
@@ -7,7 +7,7 @@
 # Find out where we are
 SCRIPTFILE=$(readlink -f "$0")
 INSTALLDIR=$(echo $SCRIPTFILE | sed -r -e 's|^(.*)/(.*)$|\1|g') #.* is greedy and eats up until the final slash
-PROG=$(command -v ipython)
+PROG=$(command -v ipython2)
```

Comment:
You can `if [ $(command -v ipython2) ]; then` around this and fall through to plain `ipython` in the else clause.